packages/react/src/runtime/sheet.ts

```diff
@@ -114,15 +127,25 @@ export const getStyleBucketName = (sheet: string): Bucket => {
     return 'm';
   }
 
+  const firstBracket = sheet.indexOf('{');
+
   /**
    * We assume that classname will always be 9 character long,
-   * using this the 10th character could be a pseudo declaration.
+   * using this the 10th characters could be a pseudo declaration.
    */
   if (sheet.charCodeAt(10) === 58 /* ":" */) {
     // We send through a subset of the string instead of the full pseudo name.
     // For example `"focus-visible"` name would instead of `"us-visible"`.
     // Return a mapped pseudo else the default catch all bucket.
-    return pseudosMap[sheet.slice(14, sheet.indexOf('{'))] || '';
+    const mapped = pseudosMap[sheet.slice(14, firstBracket)];
+    if (mapped) return mapped;
+  }
+
+  const property = sheet.slice(firstBracket + 1, sheet.indexOf(':', firstBracket)).trim();
+
+  const shorthandDepth = getShorthandDepth(property);
+  if (shorthandDepth) {
```

Comment:
`getShorthandDepth('all') === 0` which fails this logical step

⚠️ This must be changed to fix `all` bucketing (I'm pretty sure I had this bug originally as well), I just noticed in the test `packages/react/src/runtime/__tests__/style.test.tsx`
```suggestion
  if (shorthandDepth !== null) {
```
or
```suggestion
  if (typeof shorthandDepth === 'number') {
```
Sanity check: `shorthandDepth` is null | Number` right?  Both would work, but typeof would be a bit more defensive.

Reply:
nice catch! i had forgotten to update this after i changed `root` to `0`